### PR TITLE
feat: update elasticsearch to @elastic/elasticsearch@6

### PR DESCRIPTION
**Pull request by LuqmanHakim-Grune**
- change key for `host` to `node` for @elastic/elasticsearch Client initiation
- add ssl configuration on Client init
- change `httpAuth` to `auth` and it's parameter format
- make client.indices.exists() to destructuring variable assignment
- make stats() to destructuring variable assignment
- make getAlias() to destructuring variable assignment
- make .search() to destructuring variable assignment

- Investigated the search feature can only working while using old index
- Investigated indexing not working
- Compare @elastic/elasticsearch `index status` and `index meta` with elasticsearch.js
- Check and debug the client API result on elasticsearch.ts
- Research writing index data flow
- Bring back requestTimeout on client initiation
- Remove requestTimeout on bulkWrite index

### CODE REVIEW

File: packages/app/src/server/service/search-delegator/elasticsearch.ts [hunk 1]
```diff
@@ -91,9 +91,14 @@ class ElasticsearchDelegator implements SearchDelegator<Data> {
...
     const { host, httpAuth, indexName } = this.getConnectionInfo();
+    const httpAuthArr = httpAuth.split(':');
+    const username = httpAuthArr[0];
+    const password = httpAuthArr[1];
+
     this.client = new elasticsearch.Client({
-      host,
-      httpAuth,
+      node: host,
+      ssl: { rejectUnauthorized: false }, // TODO: set ssl from global env config
```
Comment (yuki-takei) [inline packages/app/src/server/service/search-delegator/elasticsearch.ts:100]: Please make a new task to fix this TODO.

**Review by yuki-takei — CHANGES_REQUESTED** [review]

File: packages/app/src/server/service/search-delegator/elasticsearch.ts [hunk 2]
```diff
@@ -91,9 +91,14 @@ class ElasticsearchDelegator implements SearchDelegator<Data> {
 
   initClient() {
     const { host, httpAuth, indexName } = this.getConnectionInfo();
+    const httpAuthArr = httpAuth.split(':');
+    const username = httpAuthArr[0];
+    const password = httpAuthArr[1];
```
[the remark this answers is not in the document]
Comment (LuqmanHakim-Grune) [reply packages/app/src/server/service/search-delegator/elasticsearch.ts]: @yuki-takei I've changed this, please check my new commit.

File: packages/app/src/server/service/search-delegator/elasticsearch.ts (hunk 1 above)
Comment (LuqmanHakim-Grune) [reply packages/app/src/server/service/search-delegator/elasticsearch.ts]: @yuki-takei actually ssl.rejectUnauthorized config does exist on our last elasticsearch, I thought it was new parameter that needs to add environment configuration.

Reference:
https://github.com/elastic/elasticsearch-js/issues/1208

Comment (LuqmanHakim-Grune) [reply packages/app/src/server/service/search-delegator/elasticsearch.ts]: last versions rejectUnauthorized default value is false, current default value is true, so we need to put rejectUnauthorized manually.

**Review by yuki-takei — APPROVED** [review]

**Pull request merged**